WebApplication/Properties/launchSettings.json

```diff
@@ -17,11 +17,11 @@
     },
     "forge-configurator-inventor": {
```

Comment:
This is shared configuration for site debugging.  Create a new one, if you need HTTP+"no browser"

Reply:
Should be fixed now :)